test/new-e2e/pkg/environments/eks.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package environments
+
+import (
+	"github.com/DataDog/test-infra-definitions/resources/aws"
+	"github.com/zorkian/go-datadog-api"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner/parameters"
+)
+
+// EKS is an environment that contains a EKS cluster, FakeIntake and Agent configured to talk to each other.
+type EKS struct {
```

Comment:
I do not think we need an `EKS` specific environment, we should be able to use a generic `Kubernetes` environment for it. It should be already defined here: https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/pkg/environments/kubernetes.go#L13

Reply:
I updated the code and reuse the kubernetes environment